### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by mootz12**

### CODE REVIEW

File: user-docs/escrowing.md [hunk 1]
```diff
@@ -8,48 +8,34 @@ Escrowing is locking [YBX tokens](ybx-tokens/) for a period of time in exchange
 
 ## Why would I escrow?
 
-YieldBlox users escrow their [YBX tokens](ybx-tokens/) for veYBX which have the ability to vote on YieldBlox [governance proposals](governance.md) which includes voing on YBX issuance allocations. In the future protocol, fees may also be paid to YBX escrowers. Basically, users can earn YBX tokens passively and control protocol development-- just for having YBX.
+YieldBlox users escrow their [YBX tokens](ybx-tokens/) for veYBX which have the ability to vote on YieldBlox [governance proposals](governance.md). Governance proposals can determine key protocol functionalities like upgrades, support assets, YBX allocations, and more! There is also a portion of YBX emissions that get paid out to users who escrow their YBX. Basically, users can earn YBX tokens passively and control protocol development - just for having YBX.
```
Comment (drew-boy) [inline user-docs/escrowing.md]: Weird parallelism for 'functionality'?

File: user-docs/escrowing.md [hunk 2]
```diff
@@ -8,48 +8,34 @@ Escrowing is locking [YBX tokens](ybx-tokens/) for a period of time in exchange
...
+[YBX token](ybx-tokens/) holders can escrow their YBX in exchange for veYBX. Their YBX is locked for a 3 month, 6 month, or 1 year time period in the YBX escrow pool. After the escrow lockup ends, the user can exchange their veYBX for the original amount of YBX they locked plus any YBX earned from YBX issuance or fees over the lock period.
 
-In the future, a YBX fee allocation may be toggled on. This would mean a portion of protocol fees may be used used to repurchase YBX on the Stellar DEX and send it to the YBX escrow pool. This staking pool is proportionally distributed to veYBX holders when their escrow period ends. Through this repurchase mechanism, YBX escrowers earn a portion of protocol revenue.
+The protocol also allows users to unlock any escrow locked YBX before the time period is up. However, early unlocked YBX is not eligible to collect fees earned over the time period. Further, there is a small penalty applied to avoid potential voting exploits.
```
Comment (drew-boy) [inline user-docs/escrowing.md]: escrow-locked? locked?

Comment (drew-boy) [inline user-docs/escrowing.md]: Link to penalty math?

File: user-docs/escrowing.md [hunk 3]
```diff
@@ -8,48 +8,34 @@ Escrowing is locking [YBX tokens](ybx-tokens/) for a period of time in exchange
...
-veYBX_{received} = Y * L * YBX_{escrowed}
-$$
-
-Where:\
-$$Y=$$ the yield multiplier\
-$$L=$$ the lockup period\
-$$YBX_{staked} =$$ the YBX escrowed
-
-Technical users can also stake by interacting with the YieldBlox smart contracts directly through Stellar Turrets.
+Consider the example where a user whishes to escrow lock 100 YBX for 3 months. Once they escrow the YBX through the YieldBlox protocol, their YBX is locked in a claimable balance that only the protocol can claim. They also receive 25 veYBX in their account that they can now use to vote in [governance proposals](governance.md).
```
Comment (drew-boy) [inline user-docs/escrowing.md]: escrow-lock

Comment (drew-boy) [inline user-docs/escrowing.md]: remove 'that'

Comment (drew-boy) [inline user-docs/escrowing.md]: remove 'that'

File: user-docs/escrowing.md [hunk 4]
```diff
@@ -8,48 +8,34 @@ Escrowing is locking [YBX tokens](ybx-tokens/) for a period of time in exchange
...
-
-A user will be shown how much veYBX they have unlocked. This number is the amount of veYBX the user can unlock for YBX. They will also be shown their next unlock period -- how long they have until they can unlock their next YBX escrow.&#x20;
-
-Technical users can also unlock by interacting with our smart contracts directly through Stellar Turrets.
-
-## Can I unlock early?
+A user can unlock any escrowed YBX they have through the YieldBlox protocol. They can unlock this position regardless of how recently it has been originally locked.
 
-Yes. Users can unlock early if they want to receive the original amount of YBX they escrowed. However, they will forfeit any additional YBX earned over the escrow period so far.
+If the escrowed position has completed the Lock Period associated with it, unlocking returns the full amount of YBX to the user plus any fees or emissions earned by the escrow locked YBX. These fees are captured by converting the veYBX issued by the escrowed position into YBX!
```
Comment (drew-boy) [inline user-docs/escrowing.md]: italicize?

File: user-docs/governance.md [hunk 5]
```diff
@@ -16,9 +16,9 @@ Once a user submits a proposal to change the protocol, users have three days to
 
 ## How does voting work?
 
-Users have three days to vote on proposals. They vote 'YES' or 'NO' on proposed protocol updates or changes using a voting smart contract. The number of votes each user has is equal to the amount of veYBX they hold.&#x20;
+Users have three days to vote on proposals. They vote 'YES' or 'NO' on proposed protocol updates or changes using a voting smart contract. The number of votes each user has is equal to the amount of veYBX they hold.
 
-If the vote passes (60% of votes are YES at the end of the voting period), the proposal status data entry is changed to “approved.” If it does not pass, the proposal account is deleted. A proposal vote must reach a quorum of at least 5% of outstanding governance tokens to pass.
+If the vote passes (60% of votes are YES at the end of the voting period), the proposal status data entry is changed to “approved”. If it does not pass, the proposal account is deleted. A proposal vote must reach a quorum of at least 5% of outstanding governance tokens to pass.
```
Comment (drew-boy) [inline user-docs/governance.md]: should be "approved."

File: user-docs/escrowing.md (hunk 2 above)
Comment (mootz12) [reply user-docs/escrowing.md]: Not currently documented

**Pull request merged**